edualert/statistics/tasks.py

```diff
@@ -167,3 +176,246 @@ def get_months_since_academic_calendar_start(current_calendar):
+    founded_absences = Count('absence', filter=Q(absence__is_founded=True, absence__taken_at__year=academic_year, absence__taken_at__month=month))
+
+    classes_query = StudentCatalogPerSubject.objects \
+        .filter(academic_year=academic_year, study_class__school_unit=school_unit) \
+        .values('subject_name', 'study_class__class_grade', 'study_class__class_letter',
+                'study_class__class_master__full_name') \
+        .annotate(unfounded_absences=unfounded_absences, founded_absences=founded_absences)
+
+    # transform query results
+    classes = {}
+    for row in classes_query:
+        class_name = '{} {}'.format(row['study_class__class_grade'], row['study_class__class_letter'])
+
+        # add class if it doesn't exist
+        if not classes.get(class_name):
+            classes[class_name] = {
+                'class_name': class_name,
+                'class_master': row['study_class__class_master__full_name'],
+            }
+
+        # add subjects if the don't exist
+        clazz = classes[class_name]
+        if not clazz.get('subjects'):
+            clazz['subjects'] = []
+
+        # add the subject to the list
+        clazz['subjects'].append({
+            'subject_name': row['subject_name'],
+            'founded_absences': row['founded_absences'],
+            'unfounded_absences': row['unfounded_absences'],
+        })
+
+    return classes
+
+
+def _create_report_xslx(filename, month_name, classes):
+    def set_border(cell):
+        cell.border = Border(
+            left=Side(border_style='thin', color='FF000000'),
+            right=Side(border_style='thin', color='FF000000'),
+            top=Side(border_style='thin', color='FF000000'),
+            bottom=Side(border_style='thin', color='FF000000'),
+        )
+
+    workbook = Workbook()
+    first_sheet = True
+
+    for clazz in classes:
+        class_name = clazz['class_name']
+        class_master = clazz['class_master']
+
+        if first_sheet:
+            # change the title of the first sheet
+            worksheet = workbook.active
+            worksheet.title = class_name
+            first_sheet = False
+        else:
+            worksheet = workbook.create_sheet(class_name)
+
+        # add title
+        worksheet['A1'] = 'FISA EVIDENTA ABSENTE'
+        worksheet['A1'].font = Font(name='Calibri', bold=True)
+        worksheet['A1'].alignment = Alignment(horizontal='center')
+        worksheet.merge_cells('A1:F1')
+
+        # add subtitle
+        worksheet['A2'] = 'Luna {} 2020'.format(month_name)
+        worksheet['A2'].font = Font(name='Calibri', bold=True)
+        worksheet['A2'].alignment = Alignment(horizontal='center')
+        worksheet.merge_cells('A2:F2')
+
+        # add deadline
+        worksheet['A3'] = 'Termen de predare:'
+        worksheet['A3'].font = Font(name='Calibri', bold=True)
+        worksheet.merge_cells('A3:C3')
+
+        worksheet['B4'] = '=Cap!B1'
```

Comment:
I don't see the purpose of using the Cap tab..
Also, here it should be "Data predării:"